authentication/pipeline/user.py

```diff
@@ -92,6 +93,10 @@ def create_user_via_email(
     data = strategy.request_data().copy()
     if "name" not in data or "password" not in data:
         raise RequirePasswordAndPersonalInfoException(backend, current_partial)
+    if len(data.get("name", 0)) < NAME_MIN_LENGTH:
+        raise RequirePasswordAndPersonalInfoException(
+            backend, current_partial, errors="Name must be at least 2 characters long."
```

Comment:
Adjust the error message, and this should be a list as the frontend expects one
```suggestion
            backend, current_partial, errors=["Full name must be at least 2 characters long."]
```